integration-tests/src/tests/client/resharding_v2.rs

```diff
@@ -35,62 +35,6 @@ use crate::utils::process_blocks::set_block_protocol_version;
 
 const P_CATCHUP: f64 = 0.2;
 
-/// The condition that determines if a chunk should be produced of dropped.
```

Comment:
Whether it's ok to remove those tests depends on the final decision on replayability. Once we're decided we no longer want to support old protocol versions I think we can just removed all of it all together, not only the parts with missing chunks.